server/views/pages/overview/partials/overviewTab/_goalsList.njk

```diff
@@ -56,14 +56,16 @@
       </table>
 
       {% if goal.note %}
-        {{ govukDetails({
-          summaryHtml: "View note<span class='govuk-visually-hidden'> for Goal " + goal.sequenceNumber + "</span>",
-          text: goal.note | formatMultilineText | safe,
-          attributes: {
-            "data-qa": "overview-notes-expander-" + goal.sequenceNumber
-          },
-          classes: "app-notes-expander"
-        }) }}
+        <details class="govuk-details app-notes-expander" data-qa="overview-notes-expander-{{ goal.sequenceNumber }}">
```

Comment:
As noted above, I've had to replace the GDS component `govukDetails` with its handwritten html equivalent for the same reason (ie. ensuring the `note` field on line 66 is correctly wrapped in an element containing the class)